redshift_sqlalchemy/dialect.py

```diff
@@ -19,6 +23,35 @@ class RedshiftImpl(postgresql.PostgresqlImpl):
         __dialect__ = 'redshift'
 
 
+## Constants
+
+COMPOUND_SORTKEY = 'compound'
+INTERLEAVED_SORTKEY = 'interleaved'
+
+# For parsing adsrc, e.g.: "identity"(445178, 0, '1,1'::text)
+IDENTITY_RE = re.compile(r"""
+    "identity"
+    \(
+      (?P<current>.*),\s
+      (?P<base>.*),\s
+      '(?P<seed>\d+),(?P<step>\d+)'.*
+    \)
+""", re.VERBOSE)
+
+# For constraints e.g. FOREIGN KEY(col1) REFERENCES othertable (col2)
+FK_RE = re.compile(r"""
+    ^FOREIGN KEY
+    \s*          # Arbitrary whitespace.
+    \((.*)\)     # Columns in primary table.
```

Comment:
mainly I'm complaining about the fact that `\(.*\)` will be much more efficient written as `\([^\(].*\)`